src/hooks/useBadge.ts

```diff
@@ -0,0 +1,28 @@
+import { DataKey } from 'models/DataKey'
+import { useMemo } from 'preact/hooks'
+import { useSnapshot } from 'valtio'
+import SealCredStore from 'stores/SealCredStore'
+
+export interface BaseBadgeContract {
+  type: DataKey
+  derivative: string
+  original: string
+}
+
+export default function (derivative: string) {
+  const { ledgers } = useSnapshot(SealCredStore)
+
+  return useMemo(() => {
+    for (const [ledgerName, ledger] of Object.entries(ledgers)) {
+      for (const original of Object.keys(ledger)) {
```

Comment:
You can get rid of`{}` here

Reply:
I'm not sure if it's a good idea to remove the {} in the for loop